core/src/main/java/org/frankframework/management/bus/endpoints/ServerStatistics.java

```diff
@@ -85,8 +88,8 @@ public Message<String> getServerInformation(Message<?> message) {
 		returnMap.put("applicationServer", getApplicationServer());
 		returnMap.put("javaVersion", System.getProperty("java.runtime.name") + " (" + System.getProperty("java.runtime.version") + ")");
 		Map<String, Object> fileSystem = new HashMap<>(2);
-		fileSystem.put("totalSpace", Misc.getFileSystemTotalSpace());
-		fileSystem.put("freeSpace", Misc.getFileSystemFreeSpace());
+		fileSystem.put("totalSpace", getFileSystemTotalSpace());
+		fileSystem.put("freeSpace", getFileSystemFreeSpace());
```

Comment:
You might want to consolidate the try catch here. 
```suggestion
try {
File systemDir = getSystemDir();
if(systemDir != null) {
		fileSystem.put("totalSpace", systemDir.getTotalSpace());
		fileSystem.put("freeSpace", systemDir.getFreeSpace());
} catch (SecurityException e) {
log.debug("not allowed to read fs partition size", e);
}
```

Reply:
Mondeling besproken dat het mooier is om deze ondoorbroken en duidelijk (als lijstje) te houden